Userland/Libraries/LibWeb/HTML/Worker.cpp

```diff
@@ -36,6 +36,9 @@ void Worker::visit_edges(Cell::Visitor& visitor)
     visitor.visit(m_document.ptr());
     visitor.visit(m_implicit_port.ptr());
     visitor.visit(m_outside_port.ptr());
+    visitor.visit(m_worker_realm.ptr());
+    visitor.visit(m_worker_scope.ptr());
+    visitor.visit(m_worker_window.ptr());
```

Comment:
These objects belong to a different GC heap in another entirely `JS::VM` (`m_worker_vm`). They should not be visited here.

Reply:
Ah OK, did not see the note about that